src/osd/OSD.cc

```diff
@@ -174,8 +174,12 @@
 #else
 #define tracepoint(...)
 #endif
-#ifdef HAVE_JAEGER
+
 #include "common/tracer.h"
+#ifdef HAVE_JAEGER
+ thread_local tracing::Tracer tracer(jaeger_configuration::jaeger_osd_config);
```

Comment:
the osd wasn't using a thread-local tracer before, so i don't think we should change that here